accounts/templates/logout.html

```diff
@@ -0,0 +1,7 @@
+<a id="openstax-logout" 
+   name="openstax" 
+   href="{% url 'logout' %}" 
+   id="openstax-logout">
+   <input type="button" value= "Logout you will still be logged into the Openstax OAuth2 Provider"/>
```

Comment:
This is not the behavior I just experienced. Where is this button getting rendered?
If I logout from the /admin area, I am forced to log back in (through accounts).